Source/Core/Core/HW/SI/SI_DeviceGBA.h

```diff
@@ -58,9 +56,15 @@ class CSIDevice_GBA : public ISIDevice, private GBASockServer
   void SendCommand(u32 command, u8 poll) override;
 
 private:
-  std::array<u8, 5> m_send_data{};
-  int m_num_data_received = 0;
+  enum class ConnectionState
+  {
+    AcceptCommand,
+    WaitForTransferTime,
+    ReceiveData
+  };
+
+  ConnectionState m_state = ConnectionState::AcceptCommand;
+  u8 m_cmd;
```

Comment:
`GBASockServer` also has one of those. Same goes for `m_send_data` before the change, so I'm not sure they were meant to be shared/inherited or anything (instead of being `private`)...?